src/bp/core/repositories/users.ts

```diff
@@ -115,4 +118,15 @@ export class KnexUserRepository implements UserRepository {
       .first()
       .then(result => result.qty)
   }
+
+  async getUserInfo(channel: string, userId: string): Promise<{ fullName: string; avatarUrl: string }> {
+    const { result: user } = await this.getOrCreate(channel, userId)
+    const fullName = `${user.attributes['first_name']} ${user.attributes['last_name']}`
+    const avatar = (user && user.attributes['picture_url']) || undefined
+
+    return {
+      fullName,
+      avatarUrl: avatar
```

Comment:
I kindof dislike avatarUrl when we store it as picture_url in the user attributes but that's only an opinion